packages/models/src/identity/index.ts

```diff
@@ -117,6 +117,8 @@ export class IdentityModel extends BaseModel {
 
     const result = await this.query(params);
 
+    console.log('getIdentitiesByDisplayName: ', result);
```

Comment:
Added these logs to troubleshoot another issue. Will clean up once that is fixed.